docs/examples/kops-tests-private-net-bastion-host.md

```diff
@@ -0,0 +1,379 @@
+
+```bash
+apiVersion: kops/v1alpha2
+kind: InstanceGroup
+metadata:
+  creationTimestamp: null
+  name: bastions
+spec:
+  image: kope.io/k8s-1.7-debian-jessie-amd64-hvm-ebs-2017-07-28
+  machineType: t2.micro
+  maxSize: 1
+  minSize: 1
+  role: Bastion
+  subnets:
+  - utility-us-east-1a
+```
+
+If want a H.A. setup for your bastions, modify minSize and maxSize and add more subnets. We'll do this later on this exercise.
+
+Save this and deploy the changes:
+
+```bash
+kops update cluster ${NAME} --yes
+```
+
+You will see an output like the following:
+
+```bash
+I0828 13:06:33.153920   16528 apply_cluster.go:420] Gossip DNS: skipping DNS validation
+I0828 13:06:34.686722   16528 executor.go:91] Tasks: 0 done / 116 total; 40 can run
+I0828 13:06:36.181677   16528 executor.go:91] Tasks: 40 done / 116 total; 26 can run
+I0828 13:06:37.602302   16528 executor.go:91] Tasks: 66 done / 116 total; 34 can run
+I0828 13:06:39.116916   16528 launchconfiguration.go:327] waiting for IAM instance profile "bastions.privatekopscluster.k8s.local" to be ready
+I0828 13:06:49.761535   16528 executor.go:91] Tasks: 100 done / 116 total; 9 can run
+I0828 13:06:50.897272   16528 executor.go:91] Tasks: 109 done / 116 total; 7 can run
+I0828 13:06:51.516158   16528 executor.go:91] Tasks: 116 done / 116 total; 0 can run
+I0828 13:06:51.944576   16528 update_cluster.go:247] Exporting kubecfg for cluster
+Kops has set your kubectl context to privatekopscluster.k8s.local
+
+Cluster changes have been applied to the cloud.
+
+
+Changes may require instances to restart: kops rolling-update cluster
+```
+
+This is "kops" creating the instance group with your bastion instance. Let's validate our cluster:
+
+```bash
+kops validate cluster
+Using cluster from kubectl context: privatekopscluster.k8s.local
+
+Validating cluster privatekopscluster.k8s.local
+
+INSTANCE GROUPS
+NAME                    ROLE    MACHINETYPE     MIN     MAX     SUBNETS
+bastions                Bastion t2.micro        1       1       utility-us-east-1a
+master-us-east-1a       Master  t2.micro        1       1       us-east-1a
+master-us-east-1b       Master  t2.micro        1       1       us-east-1b
+master-us-east-1c       Master  t2.micro        1       1       us-east-1c
+nodes                   Node    t2.micro        2       2       us-east-1a,us-east-1b,us-east-1c
+
+NODE STATUS
+NAME                            ROLE    READY
+ip-172-20-111-44.ec2.internal   master  True
+ip-172-20-44-102.ec2.internal   node    True
+ip-172-20-53-10.ec2.internal    master  True
+ip-172-20-64-151.ec2.internal   node    True
+ip-172-20-74-55.ec2.internal    master  True
+
+Your cluster privatekopscluster.k8s.local is ready
+```
+
+Our bastion instance group is there. Also, kops created an ELB for our "bastions" instance group that we can check with the following command:
+
+```bash
+aws elb --output=table describe-load-balancers|grep DNSName.\*bastion|awk '{print $4}'
```

Comment:
There should be a friendly DNS name setup for the bastion elb

Reply:
Never mind you did gossip

Reply:
jajjajajaa roger that !